### PR TITLE
Fix amulets and cobra crossbow attributes

### PR DESCRIPTION
**Pull request by Sh3Ns**
# Description

Amulet bug fix in enchanted werewolf amulet, enchanted theurgic amulet, time result (onDeEquip) and skill bug.
& register attirbutes cobra crossbow.

### **Actual**

This PR is to fix amulets enchanted werewolf amulet, enchanted theurgic amulet and cobra crossbow

### **Expected**

Equip and unequip enchanted werewolf amulet or enchanted theurgic amulet and the time will now decrease as expected, and cobra crossbow attributes corresponding.

## Type of change

Please delete options that are not relevant.

  - [x] Bug fix (non-breaking change which fixes an issue)
  - [x] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [x] Test A
  - [x] Test B

**Test Configuration**:

  - Server Version: Main
  - Client: 12.85
  - Operating System:  Windows 10 / Ubuntu 20.4

## Checklist

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I checked the PR checks reports
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [ ] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings
  - [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -47222,10 +47222,15 @@
 		<attribute key="weight" value="8000"/>
 	</item>
 	<item fromid="30374" toid="30392" name="RESERVED SPRITE"/>
-	<item id="30393" article="a" name="cobra crossbow">
+		<item id="30393" article="a" name="cobra crossbow">
+        <attribute key="weaponType" value="distance"/>
```
Comment (dudantas) [inline data/items/items.xml]:
```suggestion
	<item id="30393" article="a" name="cobra crossbow">
		<attribute key="weaponType" value="distance"/>
```

File: data/items/items.xml [hunk 2]
```diff
@@ -47222,10 +47222,15 @@
...
 		<attribute key="slotType" value="two-handed"/>
+        <attribute key="ammotype" value="bolt"/>
```
Comment (dudantas) [inline data/items/items.xml]:
```suggestion
		<attribute key="ammotype" value="bolt"/>
```

**Review by dudantas — CHANGES_REQUESTED** [review]
I added some suggestions to fix the indentation. It's just commit.

**Comment by majestyotbr** [comment]
Update your branch and commit the suggested changes.

**Review by majestyotbr — APPROVED** [review]

**Review by dudantas — APPROVED** [review]

**Pull request merged**